ibutsu_pod.yaml

```diff
@@ -0,0 +1,121 @@
+# use with podman play cube
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: ibutsu
+  name: ibutsu
+spec:
+  containers:
+  - name: postgres
+    image: docker.io/library/postgres:latest
+    resources: &resources-small
+      requests:
+        memory: "128Mi"
+        cpu: "250m"
+      limits:
+        memory: "512Mi"
+        cpu: "500m"
+    volumeMounts:
+    - name: ibutsu-database
+      mountPath: /var/lib/postgresql/data
+    env: &all-env
```

Comment:
I get that this is easier, but you're essentially exposing secrets to a container that doesn't need them. Hopefully no one would use this in the real world, but it does present an easy way for an attacker to get to other systems.

Reply:
yeah, this is only valid in a docker-compose style setup

unfortunately there is no way to clone them more sanely

Reply:
split into config maps